packages/browser/__tests__/Session.spec.ts

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+// The following is required by tsyringe
+import "reflect-metadata";
+import { it, describe } from "@jest/globals";
+import { mockClientAuthentication } from "../src/__mocks__/ClientAuthentication";
+import { Session } from "../src/Session";
+import { mockStorage } from "../../core/src/storage/__mocks__/StorageUtility";
+
+describe("Session", () => {
+  describe("constructor", () => {
+    it("accepts an empty config", async () => {
+      const mySession = new Session({});
+      expect(mySession.info.isLoggedIn).toEqual(false);
+      expect(mySession.info.sessionId).toBeDefined();
+    });
+
+    it("accepts no config", async () => {
+      const mySession = new Session();
+      expect(mySession.info.isLoggedIn).toEqual(false);
+      expect(mySession.info.sessionId).toBeDefined();
+    });
+
+    it("does not generate a session id if one is provided", () => {
```

Comment:
In comments and messages, ID should be capitalized:
```suggestion
    it("does not generate a session ID if one is provided", () => {
```